mica-rest/src/main/java/org/obiba/mica/study/rest/StudyStatesResource.java

```diff
@@ -63,24 +67,25 @@
   @GET
   @Path("/study-states")
   @Timed
-  public List<Mica.StudySummaryDto> list(@QueryParam("query") String query, @QueryParam("from") @DefaultValue("0") Integer from,
-                                         @QueryParam("limit") Integer limit, @Context HttpServletResponse response) {
-    Stream<StudyState> result;
+  public List<Mica.StudySummaryDto> listCollectionStudyStates(@QueryParam("query") String query, @QueryParam("from") @DefaultValue("0") Integer from,
+                                         @QueryParam("limit") Integer limit, @QueryParam("type") @DefaultValue("collection-study") String type, @Context HttpServletResponse response) {
```

Comment:
Should not specifying the type return all studies (harmonization and collection)? Currently, not specifying it would return the collection-study states.